runtime/gc_glue_java/ScavengerRootClearer.cpp

```diff
@@ -265,6 +265,14 @@ MM_ScavengerRootClearer::scavengeContinuationObjects(MM_EnvironmentStandard *env
 #endif /* JAVA_SPEC_VERSION >= 19 */
 }
 
+void
+MM_ScavengerRootClearer::iterateAllContinuationObjects(MM_EnvironmentBase *env)
+{
+	reportScanningStarted(RootScannerEntity_ContinuationObjectsComplete);
```

Comment:
it would be nice to skip this step if we know that are no Continuation objects in the system (to avoid the sync point that follows)

but we cannot rely on existing shouldScavengeContinuationObjects, which tells only if there are Continuation objects in Nursery

Reply:
although it's not that bad to leave as-is, because we enter this step only when there are jit methods to compile. so if we can do something about it, we can do it in a follow-up change

Reply:
add private helper isContinuationListEmpty() in MM_RootScanner()